### DISCUSSION TITLE
Separate CSS selectors for the right and left thumbs in RangeSlider

### DISCUSSION THREAD

**Discussion by mohammad-reza-rahbar** [mantinedev/mantine#6526] [Q&A]
I want to make a gradient slider like the following image from Inkscape:

![Screenshot 2024-07-15 035144](https://github.com/user-attachments/assets/e8436aab-5b4d-4bfe-8014-d6e0077542ad)

But because there is only one thumb selector, I cannot change the right and left thumbs colors easily. It would be nice to have separate left and right CSS selectors in such situations. Thanks in advance.

**Comment by rtivital** [comment]
For these cases, it is better to use use-move hook – https://mantine.dev/hooks/use-move/

**Reply by mohammad-reza-rahbar** [reply]
I did not know that. Thank you very much.
